### DISCUSSION TITLE
Change color of a single button

### DISCUSSION THREAD

**Discussion by bosch-matthias** [Akascape/CTkMessagebox#32] [Q&A] [answered]
Hey,
is it possible to change the color of a single button if there are more than one buttons?
If not that would be pretty neat as a feature.

**Comment by Akascape** [comment, marked as answer]
@bosch-matthias Yes, just pass `button_color=(color1, color2, color3)`

**Reply by bosch-matthias** [reply]
okay nice, thanks a lot!
